app/uploadform.js

```diff
@@ -0,0 +1,51 @@
+define(['jquery','app/messages', 'app/config', 'parsley', 'bootstrap-select', 'app/upload'], function ($,messages,config, parsley, select, upload) {
+  console.log(upload);
+  $(function () {
+    $(".selectpicker").selectpicker();
+    window.Parsley.on('field:error', function() {
+        if (this.$element.is("select")) {
+            this.$element.parent().children('.selectpicker').selectpicker('setStyle', 'alert-danger').selectpicker('refresh');
```

Comment:
Format these long jQuery statements better, take a look at novice to ninja for some examples.